test/playwright/test_ux.py

```diff
@@ -166,7 +166,7 @@ def _existing_environment_interactions(page, env_name, time_to_build_env=3*60*10
     # edit existing environment throught the YAML editor
     page.get_by_role("button", name=env_name).click()
     page.get_by_role("button", name="Edit").click()
-    page.get_by_label("Switch to YAML Editor").check()
+    page.get_by_label("Specification").check()
```

Comment:
```suggestion
    page.get_by_label("YAML").check()
```
There are some other changes that need to made as well. Those are being worked on in #343